tests/conftest.py

```diff
@@ -2142,9 +2151,12 @@ def backingstore_factory(request, cld_mgr, mcg_obj, cloud_uls_factory):
             a backingstore
```

Comment:
Add to the docstring when it returns None

Reply:
Done